packages/field-base/test/input-field-mixin.test.js

```diff
@@ -168,6 +168,17 @@ const runTests = (defineHelper, baseMixin) => {
       expect(spy.calledOnce).to.be.true;
     });
 
+    it('should not validate on blur when document does not have focus', () => {
+      sinon.stub(document, 'hasFocus').returns(false);
+
+      const spy = sinon.spy(element, 'validate');
+      input.focus();
+      input.blur();
+      expect(spy.called).to.be.false;
+
+      document.hasFocus.restore();
```

Comment:
If the assertion fails, the stub won't be restored and will potentially affect the following tests. It's currently not a big deal because this is the only test that relies on `hasFocus`

Reply:
Good point, thanks. I was hesitant whether to create a separate suite or not. Let me add one.